src/main/webapp/js/userMapData.json

```diff
@@ -0,0 +1,787 @@
+["Wescale","Germany"],
+["City College","Greece"],
+["MIO-ECSDE","Greece"],
+["Colegio Americano del Sur","Guatemala"],
+["HKU SPACE Community College","Hong Kong"],
+["Queen Elizabeth School","Hong Kong"],
+["The Hong Kong Polytechnic University","Hong Kong"],
+["University of Hong Kong","Hong Kong"],
+["All India Institute of Medical Sciences","India"],
+["BITS Pilani","India"],
+["Charotar University of Science & Technology","India"],
+["Gujarat University","India"],
+["Guru Gobind Singh Indraprastha University","India"],
+["IIIT","India"],
+["IIT Madras","India"],
+["Indian Institute of Technology","India"],
+["Keshav Memorial Institute of Technology","India"],
+["Maharaja's College Ernakulam","India"],
+["Manipal Academy of Banking","India"],
+["Marwadi Education Foundation","India"],
+["Nabakrushna Choudhury Centre for Development Studies","India"],
+["Pearl Academy","India"],
+["Tech Mahindra","India"],
+["VIT University Vellore","India"],
+["Bandung Institute of Technology","Indonesia"],
+["Indonesia University of Education","Indonesia"],
+["ITB School of Business and Management","Indonesia"],
+["Petra Christian University","Indonesia"],
+["Saint John's Catholic School","Indonesia"],
+["Tarumanagara University","Indonesia"],
+["Universitas Pelita Harapan","Indonesia"],
+["Shairf University of Technology","Iran"],
+["Cork Institute of Technology","Ireland"],
+["Dorset College Dublin","Ireland"],
+["Dublin Institute of Technology","Ireland"],
+["Dundalk Institute of Technology","Ireland"],
+["National University of Ireland Galway","Ireland"],
+["University College Dublin","Ireland"],
+["University of Limerick","Ireland"],
+["Hebrew University","Israel"],
+["The Kibbutzim College of Education","Israel"],
+["Arxo","Italy"],
+["Instilla","Italy"],
+["Kefato University of Trento","Italy"],
+["Università Cattolica - Milano","Italy"],
+["University of Trento","Italy"],
+["Showa Women's University","Japan"],
+["Spire","Kenya"],
+["Lebanese American University","Lebanon"],
+["Institute for Tourism Studies","Macau"],
+["Kolej PolyTech Mara","Malaysia"],
+["Sekolah Menengah Kebangsaan Bukit Jalor","Malaysia"],
+["Tunku Abdul Rahman University","Malaysia"],
+["UCSI University","Malaysia"],
+["Universiti Kebangsaan Malaysia","Malaysia"],
+["Universiti Malaysia Sarawak","Malaysia"],
+["Universiti Putra Malaysia","Malaysia"],
+["Universiti Teknologi Malaysia","Malaysia"],
+["University College of Technology Sarawak","Malaysia"],
+["University Kuala Lumpur","Malaysia"],
+["University of Nottingham Malaysia Campus","Malaysia"],
+["Escuela Bancaria y Comercial","Mexico"],
+["ITESM","Mexico"],
+["ITESO","Mexico"],
+["Peterson Schools","Mexico"],
+["Universidad Autónoma de Baja California","Mexico"],
+["Energized.Org","Netherland"],
+["HAN University of Applied Sciences","Netherland"],
+["Hogeschool van Amsterdam","Netherland"],
+["Inholland University of Applied Sciences","Netherland"],
+["Kaj Munk College","Netherland"],
+["VU University","Netherland"],
```

Comment:
I don't think we need another 'Netherland'. Also, I believe the Hogeschool up there is the same as the one down.